pinot-clients/pinot-java-client/src/test/java/org/apache/pinot/client/ExternalViewReaderTest.java

```diff
@@ -75,7 +75,7 @@ public void testGetLiveBrokersExceptionState() throws IOException {
     // Setup
     final List<String> expectedResult = Arrays.asList();
     when(mockZkClient.readData(Mockito.anyString(), Mockito.anyBoolean())).thenThrow(
-        IOException.class);
+        RuntimeException.class);
```

Comment:
Is this related?

Reply:
This is caused by type inference check in java 11 as ZkClient.readData() doesn't throw IOException, so the mock cannot throw it as well.